.github/workflows/release.yml

```diff
@@ -0,0 +1,56 @@
+name: Build and release to PyPI
+on:
+  deployment: {}
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      PYTHON_VERSION: "3.7"
+    strategy:
+      matrix:
+        node: [12.x]
+    steps:
+    - uses: actions/checkout@v1.1.0
+      with:
+        token: ${{ secrets.flexgetbot_pat }}
+    - name: Setup Git User
+      run: |
+        git config user.email ${{ secrets.git_email }}
+        git config user.name ${{ secrets.git_user }}
+    - name: Set up Python ${{ env.PYTHON_VERSION }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ env.PYTHON_VERSION }}
+      # TODO: Uncomment this when supported: https://github.com/actions/cache/issues/63
```

Comment:
WOW this was fixed yesterday 🙏 
https://github.com/actions/cache/pull/263
https://github.com/actions/cache/issues/63

Reply:
i'm going to just wait until the v2 releases before using this functionality but i'll change the comment